.circleci/config.yml

```diff
@@ -433,6 +480,23 @@ workflows:
             tags:
               only: /.*/
 
+      # Internationalization jobs
+      #
+      # Extract strings and upload them to our translation management platform
+      - build-i18n:
+          requires:
+            - build-back
+          filters:
+            tags:
+              only: /.*/
+      - upload-i18n-strings:
+          requires:
+            - build-back
+            - build-i18n
+          filters:
+            branches:
+              only: main
+
```

Comment:
did you forget `upload-i18n-strings`?

Reply:
Sorry I did not understand your point ?

Reply:
sorry I didn't read properly :hand_over_mouth: